src/cloudimagedirectory/connection/connection.py

```diff
@@ -0,0 +1,87 @@
+import json
+import os
+import pathlib
+
+from pathlib import Path
+
+
+exception_not_connected = Exception("Client is not connected to s3 bucket")
+exception_already_connected = Exception("Client is already connected to s3 bucket")
+exception_path_not_existing = Exception(
+    "The given path to the filesystem doesn't exist"
+)
+exception_not_implemented = Exception("Not implemented")
+
+
+class Connection:
+    def connect(self):
+        pass
+
+    def get_filenames(self):
+        pass
+
+    def get_content(self, filename):
+        pass
+
+    def put_content(self, content, filename):
+        pass
+
+
+class DataEntry:
+    filename = ""
+    content = None
+
+    def __init__(self, filename, content):
+        self.filename = filename
+        self.content = content
+
+    def to_bytes(self):
```

Comment:
Interesting function. Could you please explain what it is used for?

Reply:
Oh yes, this function is not longer needed :) . I must have overlooked it - thank you :smiley: